### ISSUE TITLE
Code Table Request - definitions for part preservations

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#5689]
**How to Use This Form**
I recently got back some small mammal clavicles. They were embedded in plastic, cut in two, and a couple of thin section slides made from the block. I am trying to work through what to call these new pieces. I see there are a bunch of options for the slides. I can check to see what exactly the plastic is from the loanee, but is there a general term for "clear plastic block"
<img width="214" alt="Screenshot 2023-02-21 at 10 18 09 AM" src="https://user-images.githubusercontent.com/5749672/220438838-d7936e8b-6d45-487c-96c4-31269ae947f7.png">
 in preservation?

**Goal**
To describe parts newly acquired.

See attached list of come currently available parts. There does not seem to be a link to definitions.

**Comment by Jegelewicz** [comment]
Current "slide preservations" include

Part_Preservation | Tissueness | Description 
-- | -- | -- 
slide [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#slide) ] | No Influence | a thin flat piece of glass, typically 75 by 26 mm (3 by 1 inches) and about 1 mm thick but dimensions may vary
slide, Cobb metal [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#slide__cobb_metal) ] | No Influence | two cover glass slides mounted in a thin metal carrier
slide smear [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#slide_smear) ] | No Influence | a thin layer of blood, fecal material, or other sample spread  across the surface of a glass slide and fixed or allowed to dry
slide, thin section [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#slide__thin_section) ] | No Influence | A thin (usually sub-millimeter) cross-section of a specimen adhered to the surface of a glass slide.

> is there a general term for "clear plastic block"

Not now. The closest thing I can see is

Part_Preservation | Tissueness | Description 
-- | -- | -- 
paraffin-embedded [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#paraffin_embedded) ] | Allows | a waxy solid consisting of a mixture of saturated hydrocarbons,  obtained by distillation from petroleum, typically used to as an  embedding medium for tissue sections fixed in formalin

which may be exactly what you have or may not.

Part_Preservation | Tissueness | Description 
-- | -- | -- 
sectioned [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#sectioned) ] | No Influence | made or divided into a separate sections
thin section [ [link](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#thin_section) ] | No Influence | A thin cross-section of a specimen

also exist and if the thin sections aren't slide mounted, perhaps one of these would be useful?

**Comment by ewommack** [comment]
Maybe call it the bone and then add a new preservation? Some new type of embedded material, like there is [paraffin-embedded](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation#paraffin_embedded) for preservation?

**Comment by cjconroy** [comment]
I think the part will be bone for sure. And, for the slides, I will use "slide, thin section". It would be helpful if there was a link to these definitions somewhere near the Preservation Attributes dropdown. It's not obvious to me where I would find those definitions. 
Not clear to me what the simple  "slide" is as a form of preservation. It describes a piece of glass, but not how it is used.
The block I have is super hard resin, not paraffin. Waiting for the researcher to tell me what kind it is. Maybe a term like "embedded" would be a good general preservation term, and resin could go in a remark.

**Comment by Jegelewicz** [comment]
> part will be bone for sure.

I'd suggest being more specific. You said `some small mammal clavicles` so I would use [clavicle](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#clavicle). It needs to be added to Mamm collections, but that's no big deal.

> Not clear to me what the simple "slide" is as a form of preservation. It describes a piece of glass, but not how it is used.

I think it may have been left that way so that it could be "used" in many ways - with the details provided in the attribute remark

**Comment by cjconroy** [comment]
The researcher suggested "resin-embedded" as a preservative for these clavicles. Does that also need to be added as a preservation type? I hate to add something that is only going to be used for 0.00001% of Arctos. 
Also, when I originally loaned out the clavicle, I think my only option then was as a subsample of the skeleton. If you create clavicle, can I change that part, or do I make a new part? Its the same part, but is now back in the collection. If I make a new part, then I suppose I would need to add it to the loan?

**Comment by Jegelewicz** [comment]
> can I change that part

I think so, but because it is used in a loan, maybe not. I have just added clavicle to Mamm collections - give it a while to let the change take effect, then try changing one and see if it lets you.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I was just adding some things with this same preservation the past couple of days - fossil eggshell embedded in epoxy from which thin sections had been cut. I used sectioned, but epoxy-embedded would be better.

**Comment by Jegelewicz** [comment]
@cjconroy @Nicole-Ridgwell-NMMNHS is there an actionable code table request embedded in this thread? If so, please file a separate issue using the code table request template, if not, let me know so that I can close this. Thanks!

**Comment by cjconroy** [comment]
Not from me. I just changed my parts of "bone" to "clavicle".

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
_**Goal**: Add a preservation to described embedded specimens




_**Proposed Value**: resin embedded




_**Proposed Definition**: a specimen or object embedded in resin (a substance that converts into rigid polymers by the process of curing) typically for the purposes of thin-sectioning. [Github Issue](https://github.com/ArctosDB/arctos/issues/5689)




_**Context**: Paraffin embedded exists already, but is a different material. I don't think the two should be merged under "embedded" because I think the different properties of the two substances probably warrant different treatment.




_**Table**: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation




@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)


## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._
